Tests/MongoSwiftTests/ChangeStreamTests.swift

```diff
@@ -245,6 +245,12 @@ final class ChangeStreamSpecTests: MongoSwiftTestCase, FailPointConfigured {
     }
 
     func testChangeStreamSpec() throws {
+        // TODO SWIFT-539: unskip
```

Comment:
in the situations where we did actually manage to get the server version and run these (on repl sets), this test just ended up hanging and timing out later on. so also skipping for now.